llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -1670,11 +1798,15 @@ Expected<bool> FunctionImporter::importFunctions(
       if (!GV.hasName())
         continue;
       auto GUID = GV.getGUID();
-      auto Import = ImportGUIDs.count(GUID);
-      LLVM_DEBUG(dbgs() << (Import ? "Is" : "Not") << " importing global "
-                        << GUID << " " << GV.getName() << " from "
-                        << SrcModule->getSourceFileName() << "\n");
-      if (Import) {
+      auto ImportType = maybeGetImportType(ImportGUIDs, GUID);
+      if (!ImportType)
```

Comment:
Do we need to emit a debug message in this case like you are doing for functions above? Ditto for aliases below

Reply:
Or do what I suggested above which goes back to only needing one LLVM_DEBUG